README.md

```diff
@@ -240,7 +269,8 @@ Query layer:
 - [ ] column index
 - protocol
   - [ ] gRPC
-  - [ ] MySQL
+  - [x] MySQL
+  - [x] SQLite
```

Comment:
We don't support mysql wire protocol yet, so I think we shouldn't check this box. As for sqlite, we can remove the entry because it is an embedded database, so there is no wire protocol.